static/app/views/insights/types.tsx

```diff
@@ -72,6 +72,16 @@ export enum SpanMetricsField {
   MOBILE_SLOW_FRAMES = 'mobile.slow_frames',
 }
 
+// TODO: This will be fine Field type for eap spans
```

Comment:
!?

Reply:
Ugh typing is hard,
It's meant to say "this will be the final type for EAP spans"

Reply:
Although I don't think this should be in this PR oops

Reply:
Basically after the migration we'll remove useSpanMetrics, useSpanIndexed. useMetrics, useDiscoverOrEap, and their series counterparts.

The SpanFields type will replace SpanMetricsFields, MetricsFields and SpanIndexedFields

Reply:
Roger, makes sense!